packages/ai-jsx/package.json

```diff
@@ -15,7 +15,6 @@
         "default": "./dist/esm/index.js"
       },
       "require": {
-        "types": "./dist/cjs/index.d.cts",
```

Comment:
I had some issues with `yarn build` and as part of the fix, we also removed the `.cts` files that don't actually exist.